api/SpeechRecognitionEvent.json

```diff
@@ -47,7 +47,7 @@
           },
           "webview_android": {
             "prefix": "webkit",
-            "version_added": true,
+            "version_added": "4.4.3",
```

Comment:
Since this API requires platform integration I'm not too confident about an exact number based on inference from Chrome for Android. Can we go with <=37 instead?